### DISCUSSION TITLE
Is it possible to make a NorESM-CAM / CAM split in namelist_defaults_cam.xml

### DISCUSSION THREAD

**Discussion by oyvindseland** [NorESMhub/CAM#72] [Ideas]
One of the reasons why it is a lot of namelist files or at least quite long namelist files in NorESM-CAM is that for every new namelist files a number of CAM values must be overwritten and there is a strong risk for forgetting some values when writing new namelist files. 

Is it possible to include these as an option in  namelist_defaults_cam.xml .  Potential alternative? Add a NorESM compset defaults with priority for NorESM compsets. Given the large number of backwards compability settings in namelist_defaults I imagine that it will be shortened down considerably so that the file length will not increase at the end. 
@gold2718

**Comment by gold2718** [comment]
namelist_defaults_cam.xml is now separately maintained in the [CAM/noresm](https://github.com/NorESMhub/CAM/tree/noresm_develop)_develop branch. This means that changes can be made to that file to support CAM-Nor needs. Any setting that is changed by NCAR that is also modified in the CAM-Nor version can be discussed before updating the CAM-Nor version (noresm_develop).
